internal/pkg/values/timespan.go

```diff
@@ -75,7 +75,38 @@ func (t *timeSpans) Set(value string) error {
 }
 
 func (t *timeSpans) String() string {
-	return fmt.Sprint(*t)
+	var stringBuilder strings.Builder
+
+	for i, timespan := range *t {
+		if i > 0 {
+			stringBuilder.WriteString(", ")
+		}
+
+		switch span := timespan.(type) {
+		case absoluteTimeSpan:
+			stringBuilder.WriteString(fmt.Sprintf("AbsoluteTimeSpan(%s - %s)", span.from.Format(time.RFC3339), span.to.Format(time.RFC3339)))
```

Comment:
i would change `AbsoluteTimeSpan` to the actual type name, so `absoluteTimeSpan`
```suggestion
			stringBuilder.WriteString(fmt.Sprintf("absoluteTimeSpan(%s - %s)", span.from.Format(time.RFC3339), span.to.Format(time.RFC3339)))
```